.circleci/config.yml

```diff
@@ -240,6 +240,44 @@ jobs:
           command: |
               cd driverkit
               make cleanup_s3
+  "test-infra/scan/terraform":
+    docker:
+      - image: accurics/terrascan:latest
+    steps:
+      - checkout
+      - run:
+          name: Scan terraform code
+          command: |
+            #terrascan scan -d ./config/clusters -i terraform -t aws
```

Comment:
```suggestion
            terrascan scan -d ./config/clusters -i terraform -t aws
```

Reply:
This is a max question he wrote this job, @maxgio92